runtime/doc/treesitter.txt

```diff
@@ -834,6 +834,12 @@ TSNode:range({include_bytes})                                 *TSNode:range()*
     Parameters: ~
       • {include_bytes}  (`boolean?`)
 
+    Return (multiple): ~
+        (`integer`)
+        (`integer`)
+        (`integer`)
+        (`integer`)
```

Comment:
Given that this can return 6 values, shouldn't we have:
```suggestion
    Return (multiple): ~
        (`integer`)
        (`integer`)
        (`integer`)
        (`integer`)
        (`integer|nil`)
        (`integer|nil`)
```

Reply:
There's an overload that covers that.

Reply:
Fair enough.

Also, if I understand https://github.com/neovim/neovim/blob/master/scripts/cdoc_grammar.lua correctly, we don't handle `@overload` in our docs generation right?